core/client/fs/src/main/java/alluxio/client/file/BaseFileSystem.java

```diff
@@ -683,7 +683,7 @@ private void checkUri(AlluxioURI uri) {
        * user passes. If not, throw an exception letting the user know they don't match.
        */
       Authority configured =
-          MasterInquireClient.Factory.create(mFsContext.getConf())
+          MasterInquireClient.Factory.create(mFsContext.getConf(uri))
```

Comment:
I don't think we need `uri` here since it's looking up connection details, and it doesn't make sense for those to be path-based